src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -1,25 +1,35 @@
 <template>
   <div id="filing-history-list">
-    <add-comment-dialog
+    <AddCommentDialog
       :dialog="addCommentDialog"
       :filingId="currentFilingId"
       @close="hideCommentDialog($event)"
       attach="#filing-history-list"
     />
 
-    <download-error-dialog
+    <DownloadErrorDialog
       :dialog="downloadErrorDialog"
       @close="downloadErrorDialog=false"
       attach="#filing-history-list"
     />
 
-    <load-correction-dialog
+    <LoadCorrectionDialog
       :dialog="loadCorrectionDialog"
       @exit="loadCorrectionDialog=false"
       attach="#filing-history-list"
     />
 
-    <div class="scrollable-container" style="max-height: 56rem">
+    <!-- Alternate Loading Spinner -->
+    <v-fade-transition>
```

Comment:
I added this to show while fetching documents and detail comments. I tried simply changing buttons to their loading state but it was overly complicated and didn't look right. This spinner seems to work well (IMHO) (locally anyway).